src/hotspot/share/prims/whitebox.cpp

```diff
@@ -1097,6 +1097,22 @@ bool WhiteBox::validate_cgroup(bool cgroups_v2_enabled,
 }
 #endif
 
+bool WhiteBox::is_asan_enabled() {
```

Comment:
Does we need `bool WhileBox::is_lsan_enable()` to check '--enable-lsan           enable LeakSanitizer' enable or not